ravenframework/TSA/TSAUser.py

```diff
@@ -95,6 +99,8 @@ def readTSAInput(self, spec):
     elif self.pivotParameterID not in self.target:
       # NOTE this assumes that every TSAUser is also an InputUser!
       raise IOError('TSA: The pivotParameter must be included in the target space.')
+    if len(self._tsaAlgorithms)==0:
+      print("No Segmenting algorithms were requested.")
```

Comment:
Do we have access to the messageHandler through this class? Perhaps we should `self.raiseAWarning()` here instead of print.

Reply:
not in `TSAUser`, but we have access in `SyntheticHistory.py`. will move the check to `handleInput` there